LambdaS3FileZipper/FileZipper.cs

```diff
@@ -64,32 +65,28 @@ public async Task<FileResponse> Compress(string zipFileKey, IEnumerable<FileResp
 
 		private async Task CreateFlatZip(string localDirectory, string zipPath)
 		{
-			using (var zipArchive = ZipFile.Open(zipPath, ZipArchiveMode.Create))
+			using var zipArchive = ZipFile.Open(zipPath, ZipArchiveMode.Create);
```

Comment:
Is this a new language feature?  Disposed as part of GC?

Reply:
C# 8 Feature - ["pattern-based using" and "using declarations"](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-8.0/using). Fewer nested braces for fun and profit - it implicitly gets disposed at the end of the current scope.

Reply:
Thanks @gary-lg!  I'm trying to keep two kids alive during a pandemic so I need you guys to keep me up to date with these advancements :).

Reply:
@gary-lg for the win 🙌